src/components/scenes/PluginViewScene.js

```diff
@@ -240,6 +241,38 @@ class PluginView extends React.Component<PluginProps, PluginState> {
       return
     }
     const parsedUrl = parse(navState.url, {}, true)
+    if (parsedUrl.query['edge-callback']) {
```

Comment:
This `if` should be 

`if (parsedUrl.query) {`

You are assuming the `query` parameter exists on the next line so you should check that it does.

Reply:
no longer relevant